ctoken-monitor/publish.log

```diff
@@ -0,0 +1 @@
+Mon, 21 Mar 2022 18:15:44 GMT: successfully pushed image with reference bafybeiesfylfhlogmav6eacywpzidmcl2x6cab6xr5s53rcbtrle26fwce@sha256:c4781b2ba98db27d3873b71cf89fc251e8bdbc6a6d0e390427878ff900dbd0b8
```

Comment:
We have not been committing the `publish.log` files, so this can be removed from the pull request.